### PR TITLE
Worked on the new design changes on swap

### PR DESCRIPTION
**Pull request by Sanjeev1308**
Worked on the slippage
Refactor the code

### CODE REVIEW

**Review by mohsenParvar — APPROVED** [review]

File: src/app/constants/index.ts [hunk 1]
```diff
@@ -535,3 +535,18 @@ export function isMetaPool(poolName = ""): boolean {
 }
 
 export const BaseUrl = `https://base.snowball.io`;
```
Comment (saito-sv) [inline src/app/constants/index.ts:537]: is this a valid URL?

**Review by saito-sv — APPROVED** [review]
LGTM

File: src/app/constants/index.ts (hunk 1 above)
Comment (Sanjeev1308) [reply src/app/constants/index.ts:537]: Not sure, I haven't added this, may be @mohsenParvar provide comments over it

**Comment by saito-sv** [comment]
This PR is automatically being deployed to https://preview-task-sanjeev-swap-new-changes.dev-app.axial.exchange

**Review by bigwampa — CHANGES_REQUESTED** [review]
Approve checkbox is not visible
Screen Shot 2022-03-22 at 2 11 18 PM


Slippage currently doesn't accept input. It should also deselect whichever slippage was previously selected:
Screen Shot 2022-03-22 at 2 14 50 PM

Finally, every trade route should show the expected trade amount, regardless of whether the user has a wallet connected. This route TUSD to MIM does not show either way.

Screen Shot 2022-03-22 at 2 11 53 PM

**Comment by saito-sv** [comment]
This PR is automatically being deployed to https://preview-task-sanjeev-swap-new-changes.dev-app.axial.exchange

**Comment by saito-sv** [comment]
@Sanjeev1308  can you fix the conflicts please.

**Comment by saito-sv** [comment]
This PR is automatically being deployed to https://preview-task-sanjeev-swap-new-changes.dev-app.axial.exchange

**Pull request merged**